src/operator/mshadow_op.h

```diff
@@ -27,6 +27,7 @@
 #define MXNET_OPERATOR_MSHADOW_OP_H_
 
 #include <mxnet/base.h>
+#include <algorithm>
```

Comment:
Are you using any functions in this library? if not please get rid of this line.

Reply:
done